bonus_guides/S_phoenix_behind_proxy.md

```diff
@@ -24,12 +23,28 @@ use Mix.Config
 
 config :hello_phoenix, HelloPhoenix.Endpoint,
   http: [port: {:system, "PORT"}],
-  url: [host: "example.com"],
   cache_static_manifest: "priv/static/manifest.json"
 
 . . .
 ```
 
+Urls generated using a `_url` function from the `HelloPhoenix.Router.Helpers` module will include `localhost:port_as_above`. To fix that, you need to add the following `url` option:
```

Comment:
Could we rewrite this as:

 > Urls generated using a `_url` function from the `HelloPhoenix.Router.Helpers` module will include a url such as http://localhost:8080/users for `user_url(conn, :index)`. To fix this we can use the `url` option: